### PR TITLE
Story2

### PR DESCRIPTION
**Pull request by mfk07**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -3,10 +3,18 @@
...
 $db = getDb();
-$finals = getData($db);
 
+if(isset($_POST['season']) && isset($_POST['premier']) && isset($_POST['runners-up']) && isset($_POST['score'])) {
+    $getSeason = filter_var($_POST['season'], FILTER_VALIDATE_INT);
+    $getPremier = filter_var($_POST['premier'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+    $getRunnersUp = filter_var($_POST['runners-up'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+    $getScore = filter_var($_POST['score'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+
+sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore);
```
Comment (ashleycoles) [inline index.php:14]: One extra indent here

File: functions.php [hunk 2]
```diff
@@ -24,6 +24,21 @@ function getData($db) {
     return $query->fetchAll();
 }
 
+/**
+ * Populates database with form inputs
+ *
+ * @param [type] $db
+ * @return void
+ */
+function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore) {
```
Comment (ashleycoles) [inline functions.php:33]: You can also add void as a return type on the function itself

`function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore): void {`

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Great work! Just 2 suuuuuper minor comments and then it's good to go!

File: index.php (hunk 1 above)
Comment (mfk07) [reply index.php:14]: Indent added

File: functions.php (hunk 2 above)
Comment (mfk07) [reply functions.php:33]: void return added

**Pull request merged**